src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/GlobalCatalog.cs

```diff
@@ -263,7 +263,6 @@ public ReadOnlyActiveDirectorySchemaPropertyCollection FindAllProperties()
                 {
                     throw ExceptionHelper.GetExceptionFromCOMException(context, e);
                 }
-                DirectoryContext schemaContext = Utils.GetNewDirectoryContext(Name, DirectoryContextType.DirectoryServer, context);
```

Comment:
Same question.  This doesn't have side effects we need to keep?  Our testing on this area is not good.

Reply:
Here a new context is created. Can not see anything in the codeflow which would cause any side effect if it isn't executed

Reply:
You're almost surely correct but since our testing of this library is not complete I suggest to leave it with a discard.

Reply:
added the discard